app/views/content-design/style-guide/index.html

```diff
@@ -350,6 +354,10 @@
 
 ## T
 
+### Teach First
+
+Capitalised. The name of a training provider. Often also understood by users as the name of the [high potential initial teacher training](#high-potential-initial-teacher-training) route.
```

Comment:
Would maybe add to this that though users tend to use 'Teach First' as the name of the training, we shouldn't (because it's a type of training _delivered_ by Teach First, rather than that being the name of the programme). This was something that came up when I discussed wording with policy for some content on Get Into Teaching.

Reply:
@EllieNodder agreed! Are there any suggestions from GIT on how teams can approach this?